OpenTelemetry.AutoInstrumentation.nuspec

```diff
@@ -0,0 +1,66 @@
+        <dependency id="OpenTelemetry.Exporter.Zipkin" version="1.3.1" />
+        <dependency id="OpenTelemetry.Extensions.Propagators" version="1.3.1" />
+        <dependency id="OpenTelemetry.Instrumentation.GrpcNetClient" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Http" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Runtime" version="1.0.0" />
+        <dependency id="OpenTelemetry.Instrumentation.SqlClient" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Wcf" version="1.0.0-rc.7" />
+        <dependency id="OpenTelemetry.Shims.OpenTracing" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.AspNetCore" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs" version="1.3.0-rc.2" />
+      </group>
+      <group targetFramework=".NETFramework4.6.2">
+        <dependency id="OpenTelemetry" version="1.3.1" />
+        <dependency id="OpenTelemetry.Api" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.Console" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.Jaeger" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.OpenTelemetryProtocol" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.Prometheus" version="1.3.0-rc.2" />
+        <dependency id="OpenTelemetry.Exporter.Zipkin" version="1.3.1" />
+        <dependency id="OpenTelemetry.Extensions.Propagators" version="1.3.1" />
+        <dependency id="OpenTelemetry.Instrumentation.GrpcNetClient" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Http" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Runtime" version="1.0.0" />
+        <dependency id="OpenTelemetry.Instrumentation.SqlClient" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Wcf" version="1.0.0-rc.7" />
+        <dependency id="OpenTelemetry.Shims.OpenTracing" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.AspNet" version="1.0.0-rc9.5" />
+      </group>
+    </dependencies>
+    <frameworkAssemblies>
+      <frameworkAssembly assemblyName="System.Configuration" targetFramework=".NETFramework4.6.2" />
+      <frameworkAssembly assemblyName="System.Web" targetFramework=".NETFramework4.6.2" />
+      <frameworkAssembly assemblyName="System.ServiceModel" targetFramework=".NETFramework4.6.2" exclude="Build,Analyzers" />
+    </frameworkAssemblies>
+  </metadata>
+  <files>
+    <file src="..\tracer-home\AdditionalDeps\**\*.*" target="AdditionalDeps" />
+    <file src="..\tracer-home\net462\**\*.*" target="net462" />
+    <file src="..\tracer-home\netcoreapp3.1\**\*.*" target="netcoreapp3.1" />
+    <file src="..\tracer-home\store\**\*.*" target="store" />
+    <file src="..\tracer-home\win-x64\**\*.*" target="win-x64" />
```

Comment:
Either the win-* folders are conditional or we just build the package on Windows

Reply:
We would have to build on ALL OSes and have a job that merges it into one...

Reply:
@pellared Do you have any thoughts on how to get it done?

Reply:
For sure something like this will be needed: https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts#passing-data-between-jobs-in-a-workflow

Also, I suggest putting any logic in a Nuke target. No more thoughts at the moment.

Reply:
The most tricky part will be probably pack together alpine and linux native dependencies.

Reply:
Thanks @pellared. Your inputs has helped to build a package based on git workflow. Now, NuGet has files from all Windows, Ubuntu and Mac-OS.

![image](https://user-images.githubusercontent.com/9479006/194808081-6e3322d1-0811-4779-86b2-c7931ece929b.png)

@pjanotti FYI.